src/wasm/wasm.cpp

```diff
@@ -891,6 +891,8 @@ void TableInit::finalize() {
   }
 }
 
+void ElemDrop::finalize() {}
```

Comment:
We have to make sure that the type gets initialized somewhere, and this is the natural place.

```suggestion
void ElemDrop::finalize() { type = Type::none; }
```